libsolidity/formal/SMTLib2Interface.cpp

```diff
@@ -79,7 +79,7 @@ void SMTLib2Interface::declareFunction(string _name, vector<Sort> const& _domain
 			"| (" +
 			domain +
 			") " +
-			(_codomain == Sort::Int ? "Int" : "Bool") +
+			(_codomain == Kind::Int ? "Int" : "Bool") +
```

Comment:
Hm shouldn't this be `_codomain.kind`?

Reply:
Yes

Reply:
Why did it compile though?

Reply:
Probably the `Sort` constructor that takes a `Kind`.